Projects/TEAM-1/Turbal Y/final/final/Program.cs

```diff
@@ -0,0 +1,221 @@
+            b = (Transport)y;
+            return string.Compare(a.kind, b.kind, StringComparison.Ordinal);
+        }
+    }
+    class CompInv2 : IComparer
+    {
+        // Реалізовано интерфейс IComparer.
+        public int Compare(object x, object y)
+        {
+            Transport a, b;
+            a = (Transport)x;
+            b = (Transport)y;
+            return string.Compare(a.vehicle, b.vehicle, StringComparison.Ordinal);
+        }
+    }
+
+    class CompInv3 : IComparer
+    {
+        // Реалізовано интерфейс IComparer.
+        public int Compare(object x, object y)
+        {
+            Transport a, b;
+            a = (Transport)x;
+            b = (Transport)y;
+            return string.Compare(a.brend, b.brend, StringComparison.Ordinal);
+        }
+    }
+    class CompInv4 : IComparer
+    {
+        // Реалізовано интерфейс IComparer.
+        public int Compare(object x, object y)
+        {
+            Transport a, b;
+            a = (Transport)x;
+            b = (Transport)y;
+            if (a.brend == b.brend)
+                return string.Compare(a.model, b.model, StringComparison.Ordinal);
+            else
+                return 0;
+        }
+    }
+    [Serializable]
+    public class Transport
+    {
+        [XmlElement()]
+        public string kind;
+        [XmlElement()]
+        public string vehicle;
+        [XmlElement()]
+        public string brend;
+        [XmlElement()]
+        public string model;
+        [XmlElement()]
+        public int year_of_production;
+        public Transport(string _kind, string _vehicle, string _brend,string _model,int year)
+        {
+            kind = _kind;
+            vehicle = _vehicle;
+            brend = _brend;
+            model = _model;
+            year_of_production = year;
+
+        }
+        public Transport()
+        { }
+        public override string ToString()
+        {
+            return
+            String.Format("Kind: {0} Vehicle : {1} Brand: {2} Model: {3} Year: {4} ",
+            kind, vehicle, brend, model, year_of_production);
+        }
+    }
+    class Program
+    {
+        #region SerializationExamples
+
+        //серіалізація у XML
+        public static async void XmlSerializationExample(List<Transport> tCollection)
+        {
+            Console.WriteLine($"---------------XMLSerializationExample----------------");
+
+            XmlSerializer serializer = new XmlSerializer(typeof(List<Transport>));
+            Stream fileStream = new FileStream("e:\\tCollection.xml", FileMode.Create);
+            try
+            {
+                serializer.Serialize(fileStream, tCollection);
+                fileStream.Close();
+                Console.WriteLine("Serialization completed");
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message);
+            }
+
+            Console.WriteLine("You can check serealized XML file.  Press ENTER to continue...");
```

Comment:
Why do you use three different languages in your code and comments? Please try to use only one.